pkg/resources/manager/manager.go

```diff
@@ -472,6 +431,18 @@ func (r *resourceManager) getMachineClassAvailibilityAsString() string {
 	return removeSeparatorFromEnd(result)
 }
 
+func (r *resourceManager) getResourceList() core.ResourceList {
```

Comment:
name of function is very general. It will always return available resources.

Reply:
changed